src/psyclone/psyir/backend/sympy_writer.py

```diff
@@ -293,6 +300,11 @@ def _create_type_map(self, list_of_expressions, identical_variables=None):
         for reserved in SymPyWriter._RESERVED_NAMES:
             self._symbol_table.new_symbol(reserved)
 
+        # Set-up whether we should assume all Symbols are positive.
+        assumptions = dict()
```

Comment:
pylint asks for `{}` instead of `dict()`

Reply:
Done.